lib/matplotlib/tests/test_axes.py

```diff
@@ -6614,6 +6615,31 @@ def test_zoom_inset():
         axin1.get_position().get_points(), xx, rtol=1e-4)
 
 
+@image_comparison(['inset_polar.png'], remove_text=True, style='mpl20')
+def test_inset_polar():
+    _, ax = plt.subplots()
+    axins = ax.inset_axes([0.5, 0.1, 0.45, 0.45], polar=True)
```

Comment:
```suggestion
    axins = ax.inset_axes([0.5, 0.1, 0.45, 0.45], projection='polar')
```

given the above discussion?

Reply:
Would that also mean that [this one](https://github.com/matplotlib/matplotlib/blob/80e672e0700fa8a268aed1bdefedbd0e493e91a7/lib/matplotlib/tests/test_axes.py#L6631-L6635) is redundant and should be removed?